tekton/mario-bot/mario-image-build-trigger.yaml

```diff
@@ -145,8 +145,66 @@ spec:
         mario.bot/pull-request-id: $(tt.params.pullRequestID)
     spec:
       serviceAccountName: mario-releaser
-      pipelineRef:
-        name: clone-and-build
+      pipelineSpec:
```

Comment:
pretty much a nit: i was trying to figure out why this creates a PipelineRun that kicks off a custom task vs directly creating a Run (which is done in mario-github-comment.yaml) and im guessing it's b/c the pipeline was defined inline here and the Run doesn't support that - an alternative would be to define the pipeline as a separate type and reference it here, then you don't need the wrapping pipelinerun

anyway nbd!

Reply:
the reason here is actually that PipelineToTaskRun doesn't support finally tasks, so I needed to create a pipeline to wrap the PipelineToTaskRun tasks + the finally task